### ISSUE TITLE
Isssue with additionalSetup.ps1

### ISSUE THREAD

**Issue by gintuoklis** [Microsoft/navcontainerhelper#357]
According to the documentation additionalSetup.ps1 is reserved for the users creating container. 
[NavContainerHelper.md](https://github.com/Microsoft/navcontainerhelper/blob/master/NavContainerHelper.md)

> The default script is empty and does nothing. If you override this script there is no need to call the default behavior.

But if I look at your source code New-NavContainer.ps1, you modify this file as well (lines 612, 646)
[New-NavContainer.ps1](https://github.com/Microsoft/navcontainerhelper/blob/58edc7ae68aeac56dc43cb748c80aaee8fe90f2c/ContainerHandling/New-NavContainer.ps1)

It brings some issues when you create new container using **includeCSIDE** & **myScripts** parameters.

```
New-NavContainer -accept_eula `
                 -containerName $containerName `
                 -imageName $imageName `
                 -additionalParameters @("--volume $lsinstallationdir\:$localpathoncontainer","--env bakfile=$backupfile") `
                 -includeCSIDE `
                 -doNotExportObjectsToText `
		 -myScripts @($additionalScripts) `
                 -useBestContainerOS `
                 -licenseFile $licensefile `
                 -shortcuts Desktop 
```
It results some funky errors at the very end of container creation:
![image](https://user-images.githubusercontent.com/16342128/53176001-76e8cb80-35ed-11e9-9cab-120291abf2fe.png)

Solution for that error is to add empty line at the end of our additionalSetup.ps1. Otherwise it blurs our script, and yours scripts into one line and causes this error.

Hope it helps for someone. Wasted quite some time trying to figure where does **waitif** is coming from :)

Regards, 
Gintautas

**Comment by freddydk** [comment]
It is correct that the container leaves AdditionalSetup blank.
NavContainerHelper is a module which helps you use containers. That will use the various features of the containers in order to do its thing.
Among other things it appends lines to AdditionalSetup.ps1
It looks like your script doesn't end with a line termination - but ends with the parameter wait.
Then NavContainerHelper appends an if statement and you get the result you see.
Just make sure that your script override ends with a line terminator - then you should be fine.
If you want to avoid these things, you can use the containers without NavContainerHelper.

**Comment by gintuoklis** [comment]
Would it be more sense that you add the blank line in New-NavContainer.ps1 (lines 612, 646)? 
Otherwise, documentation should be updated that AdditionalSetup.ps1 must end with blank line :)

**Comment by freddydk** [comment]
Sure, I can do that.

**Comment by freddydk** [comment]
Empty lines in the next version:-)

**Comment by gintuoklis** [comment]
Thanks Freddy

**Comment by freddydk** [comment]
Shipped in 0.5.0.5
